page-objects/pages/LRNSTRLernStorePage.js

```diff
@@ -1 +1,117 @@
-/*[url/content]*/
\ No newline at end of file
+/*[url/content]*/
+'use strict';
+// timeout for search: 3 sec
+const waitHelpers = require('../../runtime/helpers/waitHelpers.js');
+const apiHelpers = require('../../runtime/helpers/APIhelpers');
+const stringHelpers= require('../../runtime/helpers/stringHelpers');
+const {CLIENT} = require("../../shared-objects/servers");
+const url= `${CLIENT.URL}/content/?inline=1&isCourseGroupTopic=true`;
+
+
+const selectors = {
+    searchField: ".input-active.search__container > input",
+    searchIcon: ".search__container--icon",
+    contentContainer: ".content__container",
+    contentContainer2: ".grid",
+    numberOfContentOnGUI: ".content__total",
+    firstElement: "div.content__container > div > section:nth-child(1)",
+    titleOfMaterialWhenClicked: ".content-container > .title",
+
+    courseAndTitleContainer: '.content-modal__body',
+
+    containerTopic: this.courseAndTitleContainer+'div:nth-child(2)',
+    clickableElementDropDown: '.multiselect__select',
+
+    selectorsNameOfCourses: {
+        selectorCourseTitles: "div.multiselect__content-wrapper > ul > li > span > span"
+    },
+
+    submitAddToCourseAndTopic: '[data-testid="modal_submit_btn"]',
+    selectorForMaterialLink: 'div.metadata > div:nth-child(2) > div.meta-text.text-wrap > a',
+
+
+    courseSelector: '[data-testid="courseSelector"]',
+    topicSelector: '[data-testid="topicSelector"]',
+
+
+
+
+
+}
+module.exports= {
+    selectors, 
+    switchToContentWindow: async function() {
+        let handle = await driver.getWindowHandles();
+        await driver.switchToWindow(handle[1]);
+
+    },
+    wasRedirectedToContentPage: async function() {
+        await waitHelpers.waitUntilUrlContains(url);
+    },
+    insertSearchRequest: async function(content) {
+        await waitHelpers.waitAndSetValue(selectors.searchField, content);
+        await waitHelpers.waitAndClick(selectors.searchIcon);
+    
+    },
+    rightNumberOfFoundContentDisplayed: async function(request) {
+        let selectorTotalNum = await driver.$(selectors.numberOfContentOnGUI);
+        let displayedString= await selectorTotalNum.getText();
+        let displayedNum = await stringHelpers.cutStringAfterSymbol(displayedString, ' ');
+        let apiResponse = await apiHelpers.getLernstoreMaterialsAfterRequest(request);
+        await expect(`${apiResponse}`).to.equal(displayedNum);
+    },
+    clickOnContentCard: async function(request) {
```

Comment:
The name of this method is not equal to what it does. If you need it, split it up into smaller methods